tests/ui/async-await/async-drop/live-dead-storage.rs

```diff
@@ -1,6 +1,9 @@
-//@ known-bug: #140429
+// ex-ice: #140429
 //@ compile-flags: -Zlint-mir --crate-type lib
 //@ edition:2024
+//@ build-pass
```

Comment:
since this wasn't build-fail before, it can be `check-pass` now and avoid the codegen part